libs/ui/src/screens/settings/screens/accounts-settings/hooks/use-accounts-transformer.hook.ts

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+
+import { AccountTypeEnum } from '../../../../../enums/account-type.enum';
+import { useFilteredAccounts } from '../../../../../hooks/use-filtered-accounts.hook';
+import { useAllAccountsSelector } from '../../../../../store/wallet/wallet.selectors';
+import { AccountsTransformer } from '../interfaces/accounts.interface';
+
+export const useAccountsTransformer = () => {
```

Comment:
this hook looks like overperform. 
Why we just can't fitler accounts by accounts type?